tests/NHSD.BuyingCatalogue.API.IntegrationTests/Support/ScenarioContextExtensions.cs

```diff
@@ -21,5 +22,23 @@ public static int GetCataloguePriceIdByCatalougePriceTierReference(this Scenario
 
             return cataloguePriceDictionary.TryGetValue(reference, out var cataloguePriceId) ? cataloguePriceId : 0;
         }
+
+        public static int GetCataloguePriceIdsByCatalougeSolutionId(this ScenarioContext context, string solutionId)
```

Comment:
I might be mismermembering, but you dont get it by solutionId, rather priceIdRef right?